src/app/home-page/widget/filtered-list.ts

```diff
@@ -1,7 +1,8 @@
-import { OnInit } from '@angular/core';
+import {OnDestroy, OnInit} from '@angular/core';
```

Comment:
LGTM flagged this, but I believe it's unused.

Reply:
removed unused import